homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -81,6 +87,11 @@ def make_device_data(
             devices_data.sensors.append(
                 prepare_device(hass, api, device, coordinators_by_id)
             )
+        if isinstance(device, Device) and device.device_type in ["K10+"]:
```

Comment:
Looking at the Rest API for available commands, I wonder if adding the S1, S1 Plus, K10+ and K10+ Pro can be done using the same code.

Reply:
Yes, I think so but I have only the K10+.  
We add them without testing?

Reply:
I made the change

Reply:
If the API says ut us compatible, let us try!